Appveyor: 32bit build fails

Error signature:
```
c:\qt\5.9\msvc2015\include\qtcore\qalgorithms.h(847): error C3615: constexpr function 'qCountLeadingZeroBits' cannot result in a constant expression [C:\projects\cockatrice\build\oracle\oracle.vcxproj]
  c:\qt\5.9\msvc2015\include\qtcore\qalgorithms.h(858): note: failure was caused by call of undefined function or one not declared 'constexpr'
  c:\qt\5.9\msvc2015\include\qtcore\qalgorithms.h(858): note: see usage of 'qPopulationCount'
```

This is due to the recent update of Visual Studio 2017 15.8.
Qt already knows the problem: https://bugreports.qt.io/browse/QTBUG-67259, a patch in Qt is needed to fix it: https://codereview.qt-project.org/#/c/236948/
We should hope in a quick release of the patched Qt version 5.9.7.

--- a/cmake/FindVCredistRuntime.cmake
+++ b/cmake/FindVCredistRuntime.cmake
@@ -9,7 +9,7 @@ if (WIN32)
         set(REDIST_ARCH x86)
     endif()
 
-    set(REDIST_FILE vc_redist.${REDIST_ARCH}.exe)
+    set(REDIST_FILE vcredist_${REDIST_ARCH}.exe)
 
     set(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP TRUE)
     include(InstallRequiredSystemLibraries)

--- a/cmake/NSIS.template.in
+++ b/cmake/NSIS.template.in
@@ -247,20 +247,20 @@ ${If} $PortableMode = 0
 	WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Cockatrice" "VersionMajor" "@CPACK_PACKAGE_VERSION_MAJOR@"
 	WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\Cockatrice" "VersionMinor" "@CPACK_PACKAGE_VERSION_MINOR@"
 
-	IfFileExists "$INSTDIR\vc_redist.x86.exe" VcRedist86Exists PastVcRedist86Check
+	IfFileExists "$INSTDIR\vcredist_x86.exe" VcRedist86Exists PastVcRedist86Check
 	VcRedist86Exists:
-		ExecWait '"$INSTDIR\vc_redist.x86.exe" /passive /norestart'
+		ExecWait '"$INSTDIR\vcredist_x86.exe" /passive /norestart'
 		DetailPrint "Sleep to ensure unlock of vc_redist file after installation..."
 		Sleep 3000
-		Delete "$INSTDIR\vc_redist.x86.exe"
+		Delete "$INSTDIR\vcredist_x86.exe"
 	PastVcRedist86Check:
 
-	IfFileExists "$INSTDIR\vc_redist.x64.exe" VcRedist64Exists PastVcRedist64Check
+	IfFileExists "$INSTDIR\vcredist_x64.exe" VcRedist64Exists PastVcRedist64Check
 	VcRedist64Exists:
-		ExecWait '"$INSTDIR\vc_redist.x64.exe" /passive /norestart'
+		ExecWait '"$INSTDIR\vcredist_x64.exe" /passive /norestart'
 		DetailPrint "Sleep to ensure unlock of vc_redist file after installation..."
 		Sleep 3000
-		Delete "$INSTDIR\vc_redist.x64.exe"
+		Delete "$INSTDIR\vcredist_x64.exe"
 	PastVcRedist64Check:
 
 ${Else}

